electron/main.js

```diff
@@ -116,14 +118,37 @@ const bindIpcEvents = () => {
   ipcMain.on(EVENT_TYPE.WRAPPER.RELAUNCH, lifecycle.relaunch);
 };
 
+const initWindowStateKeeper = () => {
+  const loadedWindowBounds = settings.restore(SettingsType.WINDOW_BOUNDS, {
+    height: config.WINDOW.MAIN.DEFAULT_HEIGHT,
+    width: config.WINDOW.MAIN.DEFAULT_WIDTH,
+  });
+
+  const showInFullScreen = settings.restore(SettingsType.FULL_SCREEN, undefined);
```

Comment:
Let's not set the default to undefined. This is either true or false. Maybe we should also move the getters for the individual settings to the ConfigurationPersistence and collect the defaults there instead of having them spread all over the app.

Reply:
This is again the case of the old config file. I am now setting it explicitly to 'not-set-in-v0' instead of undefined.